src/user.rs

```diff
@@ -0,0 +1,46 @@
+use std::fs::File;
+use std::io::{Read, Write};
+use std::path::{Path, PathBuf};
+
+use rustc_serialize::json;
+
+use project_types::Project;
+use Error;
+
+
+pub fn new_user<P: AsRef<Path>>(project: String, public: P, name: String) -> Result<(), Error> {
```

Comment:
I was indeed not assuming that Protonfile.json was in the present working directory. Will correct.